core/src/main/java/org/elasticsearch/search/aggregations/pipeline/movavg/models/HoltLinearModel.java

```diff
@@ -154,16 +181,17 @@ public String getName() {
         @Override
         public MovAvgModel parse(@Nullable Map<String, Object> settings, String pipelineName, int windowSize) throws ParseException {
 
-            double alpha = parseDoubleParam(settings, "alpha", 0.5);
-            double beta = parseDoubleParam(settings, "beta", 0.5);
+            double alpha = parseDoubleParam(settings, "alpha", 0.3);
+            double beta = parseDoubleParam(settings, "beta", 0.1);
             return new HoltLinearModel(alpha, beta);
         }
     }
 
     public static class HoltLinearModelBuilder implements MovAvgModelBuilder {
 
-        private double alpha = 0.5;
-        private double beta = 0.5;
+
+        private double alpha = 0.3;
+        private double beta = 0.1;
```

Comment:
Maybe use the same convention as we use in the other builders here. Have these variables as Double objects and default to null here. Then check for null before including them in the XContent output and leave defining the defaults to the parser?